tasks/main.yml

```diff
@@ -58,6 +58,22 @@
     - '{{ [ "php5-pgsql" ] if (owncloud_database in [ "postgresql" ]) else [] }}'
   tags: [ 'role::owncloud:base_install' ]
 
+- name: Create Setting Strong Directory Permissions script
+  template:
+    src: 'usr/local/sbin/secure_owncloud.sh.j2'
+    dest: '/usr/local/sbin/secure_owncloud.sh'
+    owner: 'root'
+    group: 'root'
+    mode: '0755'
+  tags: [ 'role::owncloud:base_install' ]
+  notify: ['Run Setting Strong Directory Permissions script']
+
+- name: Include www-data to owncloud_group
+  user:
+    name: www-data
+    append: yes
+    groups: "{{ owncloud_group }}"
+
```

Comment:
`./defaults/main.yml`:
```YAML
# .. envvar:: owncloud__enforce_permissions_script
#
# File path of the ownCloud permission script.
owncloud__enforce_permissions_script: '{{ (ansible_local.root.lib
                                        if (ansible_local|d() and ansible_local.root|d() and
                                            ansible_local.root.lib|d())
                                        else "/usr/local/lib") + "/owncloud-enforce-permissions" }}'
```
(That is setup by the `debops.core` role)

tasks:
```YAML
- name: Create ownCloud permission script
  template:
    src: '/usr/local/lib/owncloud-enforce-permissions.j2'
    dest: '{{ owncloud__enforce_permissions_script }}'
    owner: 'root'
    group: 'root'
    mode: '0744'
  tags: [ 'role::owncloud:base_install' ]
  notify: ['Run Setting Strong Directory Permissions script']

- name: Include www-data in owncloud_group
  user:
    name: 'www-data'
    append: True
    groups: '{{ owncloud_group }}'
```

Also maybe consider moving this section. Maybe even above the apt install. But that can be done after the thing with dpkg is figured out.